Library/Homebrew/test/dev-cmd/bump-unversioned-casks_spec.rb

```diff
@@ -0,0 +1,30 @@
+# typed: false
+# frozen_string_literal: true
+
+require "cmd/shared_examples/args_parse"
+require "dev-cmd/bump-unversioned-casks"
```

Comment:
Could this get moved into a module rather than be calling functions in `dev-cmd`?